### ISSUE TITLE
TypeError: "quotechar" must be a 1-character string

### ISSUE THREAD

**Issue by Vuizur** [LBeaudoux/tatoebatools#11]
Hello, 

thanks a lot for developing this amazing library! It is super useful to me.

I think a change to Tatoeba broke it, at least I currently get an error on my System (Windows 11, Python 3.11.1).

When executing the example code:

```python
from tatoebatools import ParallelCorpus
for sentence, translation in ParallelCorpus("cmn", "eng"):
    print((sentence.text, translation.text))
```

I get:
```
(tatoeba-to-anki-py3.11) PS C:\Programs\tatoeba-to-anki> & c:/Programs/tatoeba-to-anki/.venv/Scripts/python.exe c:/Programs/tatoeba-to-anki/tatoeba_to_anki/testing.py
downloading https://downloads.tatoeba.org/exports/per_language/cmn/cmn-eng_links.tsv.bz2
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 407k/407k [00:00<00:00, 2.86MiB/s]
decompressing cmn-eng_links.tsv.bz2
Traceback (most recent call last):
  File "c:\Programs\tatoeba-to-anki\tatoeba_to_anki\testing.py", line 2, in <module>
    for sentence, translation in ParallelCorpus("cmn", "eng"):
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\__init__.py", line 44, in __init__
    self._df = self._get_join_dataframe()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\__init__.py", line 90, in _get_join_dataframe
    links = self._get_link_dataframe()
            ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\__init__.py", line 128, in _get_link_dataframe
    return tatoeba.get("links", **params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\tatoebatools.py", line 520, in get
    self._curtable = Table(
                     ^^^^^^
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\table.py", line 87, in __init__
    self._update_required_files()
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\table.py", line 170, in _update_required_files
    update.run(verbose=self._vb)
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\update.py", line 36, in run
    self._split(downloads)
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\update.py", line 69, in _split
    dfile = DataFile(fp, **tbl_params)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Programs\tatoeba-to-anki\.venv\Lib\site-packages\tatoebatools\datafile.py", line 128, in __init__
    self._rd = csv.reader(
               ^^^^^^^^^^^
TypeError: "quotechar" must be a 1-character string
```

Have a great day!

**Comment by Vuizur** [comment]
Oh, I now understand what is failing. An empty quotechar is not allowed since Python 3.11, causing this error.

**Comment by Vuizur** [comment]
I looked a bit into it, but I haven't found out how to fix it. Setting the quote char to the default '"' causes the tests to pass, but the program example fails with:

```
Traceback (most recent call last):
  File "c:\Programs\tatoebatools\example.py", line 2, in <module>
    for sentence, translation in ParallelCorpus("cmn", "eng"):
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\__init__.py", line 44, in __init__
    self._df = self._get_join_dataframe()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\__init__.py", line 105, in _get_join_dataframe
    sentences = self._get_sentence_dataframes(row_filters)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\__init__.py", line 159, in _get_sentence_dataframes
    sentences[k] = tatoeba.get("sentences_detailed", **params)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\tatoebatools.py", line 520, in get
    self._curtable = Table(
                     ^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\table.py", line 89, in __init__
    self._dfile = self._build_datafile()
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\table.py", line 195, in _build_datafile
    return dfile.extract_rows(row_filters=self._rf)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\datafile.py", line 23, in _reset_pos
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Programs\tatoebatools\tatoebatools\datafile.py", line 297, in extract_rows
    wt.writerow(row)
_csv.Error: need to escape, but no escapechar set
```

**Comment by LBeaudoux** [comment]
@Vuizurt Thanks for reporting this issue for Python 3.11+. I will try to look into it soon.

Taoeba tab-delimited weekly exports are not quoted. Instead of using the standard `csv` module, maybe we could just read a row with `line.split("\t")`  and build a line with `"\t".join(fields)`.

**Comment by Vuizur** [comment]
> Taoeba tab-delimited weekly exports are not quoted. Instead of using the standard csv module, maybe we could just read a row with line.split("\t") and build a line with "\t".join(fields).

Sounds like a good idea.

**Comment by Fried-Squid** [comment]
I think I found a fix (Arch linux, python 3.11.1) - both escapechar and quotechar have to be single-character strings or None as of python 3.11, so adding a default for both (escapechar = None and quotechar="'") and updating config.py fixed the issue for me. See PR #13

**Comment by LBeaudoux** [comment]
@Vuizur @Rozinig @Fried-Squid thank you for your patience and for helping me solve this issue. A new [version](https://pypi.org/project/tatoebatools/0.2.2/) of tatoebatools for python 3.8+ is now available on PyPI
